src/dataaccess/GistAdjuvantIhanos.json

```diff
@@ -1299,6 +1320,13 @@
                     }
                 }
             }
+        },
+        "SourceClinicalNote": {
+            "_ShrId": "788dcbc3-ed18-470c-89ef-35ff91854c7f",
+            "_EntryId": "5200",
```

Comment:
Note 5200 is the surgical note and would be linked to the ProcedurePerformed for the surgery. The ProcedureRequested for the surgery should be linked to note 5160

Reply:
Ok I will make this change.  

- ProcedureRequested for surgery -> note 5160
- ProcedureRequested for CT-guided Biopsy -> note 5110
- ProcedureRequested for Endoscopic Biopsy -> note 5000?(Not really sure what "Order EGD with EUS" means)